File: Fundamentals of C programming/step08/8.2/8.2.06.c
/*

Для заданной квадратной матрицы A[N] посчитать следующие величины:

    сумму элементов, находящихся над главной диагональю
    сумму элементов, расположенных под побочной диагональю


Входные данные:
Одно натуральное число NN. Далее с новой строки NN строк по NN целых чисел в каждой. NN не превышают десяти.

Выходные данные:
Два целых числа, записанных через пробел. Первым вывести меньшее из чисел.

Sample Input:

3
1 2 3
1 4 5
9 3 -7

Sample Output:

1 10

*/

#include <stdio.h>
 
int main()
{
    int n = 0, a = 0, b = 0;
    int i, j;
 
    do{
        scanf("%d", &n);
      
    }while(n < 1 || n > 10);
 
    int arr[n][n];
 
    for(i = 0; i < n; i++)
    {
        for(j = 0; j < n; j++)
        {
            scanf("%d", &arr[i][j]);
            if(i < j) a += arr[i][j];
            if (i >= n - j) b += arr[i][j];
        }
    }
 
    
    (a < b) ? printf("%d %d", a, b) : printf("%d %d", b, a);
    printf("\n");
    return 0;
}

File: Diving into JavaScript: For Beginners/step08/8.02/8.2.06.js
/*

    Запросите у пользователя произвольные значения через запятую с пробелом и поместите их в массив.
    Далее, запросите у пользователя индексы элементов, которые они хотят вывести и в каком порядке через запятую.
    Выведите через пробел запрошенные элементы массива.

Sample Input 1:

1, 2, 3
0, 2, 1

Sample Output 1:

1 3 2

Sample Input 2:

11, 12, 13, 14
0, 3

Sample Output 2:

11 14

Sample Input 3:

моя, мелодия, уже, не, моя
1, 4

Sample Output 3:

мелодия моя

*/

let str = prompt();
const strArray = str.split(", ");

let num = prompt();
const numArray = num.split(", ");
const newArray = []

for (const num of numArray) {
    newArray.push(strArray[num]);
}

const res = newArray.join(' ');
console.log(res);
